dashboard/app/models/sections/lti_v1_section.rb

```diff
@@ -0,0 +1,38 @@
+# == Schema Information
+#
+# Table name: sections
+#
+#  id                   :integer          not null, primary key
+#  user_id              :integer          not null
+#  name                 :string(255)
+#  created_at           :datetime
+#  updated_at           :datetime
+#  code                 :string(255)
+#  script_id            :integer
+#  course_id            :integer
+#  grade                :string(255)
+#  login_type           :string(255)      default("email"), not null
+#  deleted_at           :datetime
+#  stage_extras         :boolean          default(FALSE), not null
+#  section_type         :string(255)
+#  first_activity_at    :datetime
+#  pairing_allowed      :boolean          default(TRUE), not null
+#  sharing_disabled     :boolean          default(FALSE), not null
+#  hidden               :boolean          default(FALSE), not null
+#  tts_autoplay_enabled :boolean          default(FALSE), not null
+#  restrict_section     :boolean          default(FALSE)
+#  properties           :text(65535)
+#  participant_type     :string(255)      default("student"), not null
+#  lti_integration_id   :bigint
+#  ai_tutor_enabled     :boolean          default(FALSE)
+#
+# Indexes
+#
+#  fk_rails_20b1e5de46        (course_id)
+#  fk_rails_f0d4df9901        (lti_integration_id)
+#  index_sections_on_code     (code) UNIQUE
+#  index_sections_on_user_id  (user_id)
+#
+
+class LtiV1Section < Section
```

Comment:
@daynew do you remember why we had to add this file? I'm pretty sure we added it during a pairing session.

Reply:
Good find, I had totally forgotten about this. Where does this code actually come into play? Trying to figure out if there's a better option that creating this empty model.

Reply:
`find_sti_class` is an override of some ActiveRecord internals. I'm not sure what a better practice would be. I don't think it would be worth our time to find a "better" solution right now.